Extension/src/LanguageServer/client.ts

```diff
@@ -830,6 +840,9 @@ export class DefaultClient implements Client {
     public lastCustomBrowseConfigurationProviderVersion: PersistentFolderState<Version> | undefined;
     private registeredProviders: PersistentFolderState<string[]> | undefined;
 
+    private configStateReceived: ConfigStateReceived = { compilers: false, compileCommands: false, configProviders: [], timeout: false };
```

Comment:
The interface allows for undefined `configProviders`. Would that be preferred over empty array?

Reply:
yeah